Source/Core/DolphinQt/Debugger/CodeDiffTableModel.cpp

```diff
@@ -0,0 +1,268 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "DolphinQt/Debugger/CodeDiffTableModel.h"
+
+#include <algorithm>
+#include <array>
+
+#include "Common/GekkoDisassembler.h"
+#include "Common/IOFile.h"
+#include "Core/PowerPC/BranchWatch.h"
+#include "Core/PowerPC/PPCSymbolDB.h"
+
+QVariant CodeDiffTableModel::data(const QModelIndex& index, int role) const
+{
+  if (!index.isValid())
+    return QVariant();
+
+  switch (role)
+  {
+  case Qt::TextAlignmentRole:
+    return TextAlignmentRoleData(index);
+  case Qt::DisplayRole:
+    return DisplayRoleData(index);
+  case UserRole::OnClickRole:
+    return OnClickRoleData(index);
+  case UserRole::SortRole:
+    return SortRoleData(index);
+  }
+  return QVariant();
+}
+
+QVariant CodeDiffTableModel::headerData(int section, Qt::Orientation orientation, int role) const
+{
+  if (orientation == Qt::Vertical || role != Qt::DisplayRole)
+    return QVariant();
+
+  static const std::array<QString, Column::NumberOfColumns> headers = {
+      tr("Instr."),      tr("Origin"),     tr("Destination"),
```

Comment:
```suggestion
      tr("Instruction"),      tr("Origin"),     tr("Destination"),
```

Since none of the others are abbreviated

Reply:
The instruction column never needs to be as wide as the full word "instruction" is.  The CodeViewWidget is the same way.